README.adoc

```diff
@@ -52,6 +52,35 @@ and configure Maven to run it after building the application.
 [role='command']
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
+[role='command']
+```
+mvn install
+```
+
+This command builds the application and creates a `.war` file in the target directory. It also
+configures and installs Open Liberty into the `target/liberty/wlp` directory.
+
+Next, run the Maven `liberty:start-server` goal:
+[role='command']
+```
+mvn liberty:start-server
+```
+
+This goal starts an Open Liberty server instance. Your Maven `pom.xml` is already configured to start
+the application in this server instance.
+
+Navigate your browser to this URL to access the application: https://localhost:9080/ServletSample/servlet[https://localhost:9080/ServletSample/servlet^]. The application displays a simple web page with the link that, when 
```

Comment:
I think this syntax https://localhost:9080/ServletSample/servlet[^] works for adding the URL with opening on a new tab action. Try it.